docs/layouts/shortcodes/generated/all_jobmanager_section.html

```diff
@@ -14,11 +14,17 @@
             <td>Integer</td>
             <td>Configure the minimum increase in parallelism for a job to scale up.</td>
         </tr>
+        <tr>
+            <td><h5>jobmanager.adaptive-scheduler.resource-stabilization-timeout</h5></td>
+            <td style="word-wrap: break-word;">10 s</td>
+            <td>Duration</td>
+            <td>The resource stabilization timeout defines the time the JobManager will wait if fewer than the required resources are available, but sufficient resources for execution are there. Once this timeout has passed, the job will start executing with the available resources, or fail, if the resources are not sufficient. The timeout starts as soon as there are sufficient resources available for execution.<br />If <span markdown="span">`scheduler-mode`</span> is configured to <span markdown="span">`REACTIVE`</span>, this configuration value will default to 0, so that jobs are starting immediately with the available resources.</td>
```

Comment:
Why are we failing if there are not sufficient resources available? I thought that this timeout would only be triggered if we have sufficient resources?

Reply:
I thought that we can not guarantee sufficient resources, even if SlotAllocator.determineParallelism says so, because we could loose slots between "sufficient resources" and "start scheduling".

Reply:
Yes, this can happen but to me the description reads a bit like we have seen once that the resources are sufficient and then we trigger the timeout and no matter what when the timeout triggers we try to schedule the job. Instead, the system should still check whether it has sufficient resources and only try to schedule the job if this is still true.

Reply:
Thanks for the clarification.